.github/workflows/formal-verification.yml

```diff
@@ -13,8 +14,16 @@ env:
   SOLC_VERSION: '0.8.19'
 
 jobs:
+  apply-diff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Apply patches
+        run: make -C certora apply
+
   list-scripts:
     runs-on: ubuntu-latest
+    # if: github.event_name != 'pull_request'
```

Comment:
Are we uncommenting this line before merging?

Reply:
yes